### PR TITLE
Finish Mobile Pull Request

### PR DESCRIPTION
**Pull request by Techcolony**
This is the Finish Mobile Pull Request to merge the finish-mobile-feature with the main branch

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -101,3 +113,306 @@ button:active {
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 88px;
+  position: absolute;
+  width: 375px;
+  height: 2788px;
+  top: 718px;
+  background: #dfe1e6;
+}
```
Comment (algerina) [inline style.css]: - [x] You did a great job however it is required to use display:grid on the main container of the work section.

File: index.html [hunk 2]
```diff
@@ -43,5 +44,198 @@ <h3 class="headline3">LET'S CONNECT</h3>
...
+    <section>
+        <div class="works">
+            <div class="card-works">
+                <div class="snapshoot-portfolio">
+                    <img src="icons-images/Snapshoot.png" alt="" />
+                </div>
+                <div class="left-block-container">
+                    <div class="left-block">
+                        <div class="project">
+                            <h1>Tonic</h1>
```
Comment (algerina) [inline index.html]: - [x] It's recommended  to use only one h1 per page for SEO reasons, kindly change h1 to h2, h3, ... on all your work cards.

File: index.html [hunk 3]
```diff
@@ -43,5 +44,198 @@ <h3 class="headline3">LET'S CONNECT</h3>
...
+                </a>
+                <div class="skill-list">
+                    <ul>
+                        <li class="drop-down">
+                            <h1>Languages</h1>
+                            <i class="fa fa-angle-down"></i>
+                        </li>
+                            <ul class="language">
+                                <li><img src="icons-images/JS.png">Javascript</li>
+                                <li><img src="icons-images/HTML.png">HTML</li>
```
Comment (algerina) [inline index.html]: - [x] It's required to use a section tag for the About section instead of a div.

File: index.html [hunk 4]
```diff
@@ -43,5 +44,198 @@ <h3 class="headline3">LET'S CONNECT</h3>
...
+                                <li><img src="icons-images/CSS.png">CSS</li>
+                            </ul>
+                        <li class="separator-border">
+                            <h1 class="separator">Frameworks</h1>
```
Comment (algerina) [inline index.html]: - [x] Kindly change this h1 to another hx tag as requested previously.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @Techcolony  ,

Great job so far! 
However, you still have minor issues to fix.

May I highlight the following: 
- [x] Your project looks awesome👏  ✔️ 
- [x] No Linter errors✔️ 


## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review.

- [ Optional] The hamburger menu and the social media icons  are not displayed kindly add a `.` at the beginning of their links.

port1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: style.css (hunk 1 above)
Comment (Techcolony) [reply style.css]: Thanks for this

File: index.html (hunk 4 above)
Comment (Techcolony) [reply index.html]: ok, that's right.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Techcolony 

Your project is complete! Great job for taking out time to make the changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- figma design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀

tufoin 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**